packages/victory-scatter/src/index.d.ts

```diff
@@ -33,10 +33,8 @@ export interface VictoryScatterProps
   origin?: OriginType;
   samples?: number;
```

Comment:
Should `samples` also be removed if it is being added to one of the common interfaces?

Reply:
Actually, are you auditing `VictoryScatter` in this PR, too?

Reply:
When I rebased my candlestick branch with master and ran `nps check.dev`. I got some errors related to the scatter, I think when @maddles worked on `VictoryScatter`, we didn't realize we had some common props that conflicted with each other's work. Therefore, I removed `sortOrder` and `range` from `VictoryScatterProps`, since I added them in the common interfaces.  I can remove the `samples` from `VictoryScatterProps` now that it is in common.

Reply:
Gotcha! makes sense.